Aqovia.PactProducerVerifier/Aqovia.PactProducerVerifier.AspNetCore.nuspec

```diff
@@ -17,6 +17,7 @@
       <dependency id="Microsoft.AspNetCore.Server.Kestrel" version="2.0.1" />
       <dependency id="Newtonsoft.Json" version="10.0.3" />
       <dependency id="PactNet.Windows" version="2.1.2" />
+      <dependency id="Microsoft.Owin.Host.HttpListener" version="4.0.0" />
```

Comment:
Why do we need to add Owin to AspNetCore project? Why can't we just use Kestrel?